resources/magento2/common.properties

```diff
@@ -22,7 +22,8 @@ common.module.target=Target Module
 common.theme.target=Target Theme
 common.area.target=Target Area
 common.name=Name
-common.className=Class Name
+common.className=Class name
+common.argument=Argument name
```

Comment:
Hey @VitaliyBoyko, @coderimus. I was just doing some local testing and I noticed that this line (line 26) with the value for `common.argument` is missing from the current codebase? This PR was merged but this particular file doesn't seem to be 'registered' in the git commit history? I'm very confused, maybe I'm missing something. Could you please check on this? Thanks.